core/src/test/resources/sql/misc.iq

```diff
@@ -2460,24 +2460,6 @@ FROM "hr"."emps";
 
 !ok
 
-!use post
```

Comment:
This test was not doing the right thing, since conformance was wrong; I replaced it with a different test, which is now in aliasing.iq